test/lib/authenticate.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+
+const test = require('ava')
+const Boom = require('boom')
+const jwt = require('jsonwebtoken')
+
+const lib = require('../../lib/authenticate.js')
+
+test('Should throw Boom error if environment variables are not set', (t) => {
+  t.plan(1)
+  try {
+    lib()
+  } catch (err) {
+    t.deepEqual(err, Boom.badImplementation('JWT_AUDIENCE environment variable is mandatory'))
+  }
```

Comment:
Is [`t.throws()`](https://github.com/avajs/ava#throwsfunctionpromise-error-message) not more appropriate here?